src/const.js

```diff
@@ -9,12 +9,29 @@ const NO_UNSANITIZED_OPTIONS = {
   escape: { taggedTemplates: [], methods: [] },
 };
 
+const NO_UNSANITIZED_METHOD_CUSTOMIZATIONS = {
+  import: {
+    escape: {
+      methods: ['chrome.runtime.getURL', 'browser.runtime.getURL'],
+    },
+    // NOTE: Alternatively using the following option would instead
+    // configure the plugin to consider any method call as allowed
+    // on dynamic import calls:
+    //
+    // objectMatches: [],
+  },
```

Comment:
@willdurand in this inline comment I'm highlighting an alternative configuration that would allow any method call to silence the no-unsanitized plugin on dynamic import calls, in the current version I have currently only added chrome.runtime.getURL and browser.runtime.getURL as valid escapers, but based on the comments on the issue side it seems we may want to also consider the alternative described in this comment (and tweak the test case accordingly).

Reply:
let's be conservative for now